index.js

```diff
@@ -439,7 +439,7 @@ class GoogleAuth extends DbBase {
       assert.ok(typeof active === 'boolean', 'active should be a boolean')
```

Comment:
can you please update the assertions here to run if the variable is defined

```suggestion
if (typeof active !== 'undefined') {
      assert.ok(typeof active === 'boolean', 'active should be a boolean')
```

Same should apply in other places where we perform similar checks.